core/src/test/java/com/coderanch/blackjack/CardsTest.java

```diff
@@ -123,4 +123,13 @@ public void getStandardDeck_returnsUnmodifiableSet() {
     public void getStandardDeck_returnsAllCombinationsOfRankAndSuit() {
         assertThat(Cards.getStandardDeck().size(), is(equalTo(Rank.values().length * Suit.values().length)));
     }
+
+    /**
+     * Tests that {@link Cards#getShuffledStandardDeck(Random)} returns a set that contains all possible distinct cards that can be
+     * made using a combination of a {@link Rank} and a {@link Suit}.
+     */
+    @Theory
+    public void getShuffledStandardDeck_returnsAllCombinationsOfRankAndSuit() {
+        assertThat(Cards.getShuffledStandardDeck(new Random()).size(), is(equalTo(Rank.values().length * Suit.values().length)));
```

Comment:
Tests must be deterministic, meaning you can't use random values.

Instead of passing a new `Random`, pass a custom subclass of `Random` that returns a deterministic sequence of values.

Let me know if you need help with this.